README.md

```diff
@@ -1 +1,200 @@
-# capstone-ebike-reservation
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
- [ ] Kindly, check the frontend [review](https://github.com/khanh-devos/capstone-ebike-reservation-frontend/pull/33#pullrequestreview-1730610704) comment.